Source/WebGPU/WebGPU/Internal/WebGPUSwiftInternal.h

```diff
@@ -51,28 +53,39 @@ using WTFRangeSizeT = WTF::Range<size_t>;
 
 __attribute__((used)) static const auto stdDynamicExtent = std::dynamic_extent;
 
+// FIXME: rdar://140819194
+constexpr unsigned long int WGPU_COPY_STRIDE_UNDEFINED_ = WGPU_COPY_STRIDE_UNDEFINED;
```

Comment:
outside the scope of this PR, but we could also just make `WGPU_COPY_STRIDE_UNDEFINED` a constexpr in WebGPU.h now that we limit the header to C++

Reply:
This hack is only needed for swift, so isn't it better living here ?